nightlies_watcher/publish.py

```diff
@@ -0,0 +1,141 @@
+import datetime
+import logging
+import os
+import re
+import taskcluster
+
+from nightlies_watcher import treeherder, hg_mozilla, tc_index, tc_queue
+from nightlies_watcher.directories import PROJECT_DIRECTORY
+from nightlies_watcher.exceptions import NotOnlyOneApkError, TreeherderJobAlreadyExistError
+
+
+logger = logging.getLogger(__name__)
+
+FENNEC_AURORA_APK_REGEX = re.compile(r'public/build/fennec-\d+.0a2.en-US.android.+\.apk')
```

Comment:
Add $ to the end of regexp maybe? This way you avoid other files, like signatures (blah.apk.asc)